frontend/rolecall/src/app/user/user-editor.component.html

```diff
@@ -12,8 +12,12 @@ <h1 class="left-card-title">
             [ngClass]="user.uuid == currentSelectedUser.uuid ? 'selected-user' : ''"
             (click)="setCurrentUser({user : user, fromInputChange : false, shouldSetLastUser : true})">
           <h2 class="user-name">
-            {{ (user.first_name ? user.first_name : " ") + " " + (user.last_name ? user.last_name
-              : " ") }}
+            {{
+              (user.first_name ? user.first_name + " " : "") + 
```

Comment:
optional: would be nice to create a pure pipe that converts a user to a full name, then the same pipe can be used both in the template and in the class.

Reply:
I like the idea! Let's do that as part of a later cleanup.